### ISSUE TITLE
Detection of Smart View Mirroring on TV

### ISSUE THREAD

**Issue by fusionstream** [jaruba/ha-samsungtv-tizen#115]
Has anyone had any luck with this? I ask because my device is being constantly update in the graph/device link of the SmartThings API but it still is forever OFFLINE. I know not why.

When I Smart Mirror, the listing in Home Assistant just shows up as my last user HDMI source (even if I come from the YouTube app to Screen Mirroring, it will show up the last used HDMI source). This suggests to me that it's an App that doesn't appear in the App List.

**Comment by jaruba** [comment]
It may not be an app at all, it may be a Tizen OS feature. Regardless, if you go in the SmartThings API Web Panel (where u created the API key) and there is no relevant information u can find in there that can identify if Smart View Mirroring is active or not, then there's nothing we can do about it.. This component can only interpret the data that is available in the SmartThings API (or the local APIs, but I'm pretty sure none of the local APIs has any info about this)

**Comment by fusionstream** [comment]
Yup. Just putting it out there in the hopes that someone else has it. Because my TV is permanently OFFLINE and I can't even get App IDs for other Apps. It's only because others like you have already documented what they've found that I've got most of what I need done.
